src/data_template_generators.js

```diff
@@ -47,7 +47,8 @@ const generateDataTemplate = (courseData, pathLookup) => ({
   ),
   open_learning_library_versions: helpers.getOpenLearningLibraryVersions(
     courseData["open_learning_library_related"]
-  )
+  ),
+  uid: courseData["uid"]
```

Comment:
Can we wrap this with `helpers.addDashesToUid`?  The UUID's that come out of the parsed Plone data are just one string as you see and standard UUID's have dashes in specific spots, this function fixes that.